### ISSUE TITLE
Focus going to background elements of "Panel" with scan mode in Narrator

### ISSUE THREAD

**Issue by saini-ankish** [OfficeDev/office-ui-fabric-react#6800]
Can see the **screen shots** here, same issue found in **[Fabric website](https://developer.microsoft.com/en-us/fabric#/components/panel)** itself 
![screenshot 2](https://user-images.githubusercontent.com/44394750/47351378-75f26c80-d6d5-11e8-9c2a-da7c17b31c85.png)

**Bug Report**
Package version(s): ">= 6.0.0 < 7"

**Browser and OS versions**: Edge and Windows 10

**Describe the issue:**
Focus going outside Panel component with narrator in scan mode.

It's also happening in the Fabric website.

**Actual behavior:**
Narrator focus going to background elements when we traverse in Scan mode using arrow keys.

**Expected behavior:**
Narrator focus should only move between first and last element inside Panel

**Comment by dzearing** [comment]
This is a duplicate of https://github.com/OfficeDev/office-ui-fabric-react/issues/6422.

**Comment by dzearing** [comment]
@aneeshack4 It seems that Panels which also are considered modal (I think there may be a setting to not make them modal though) are missing the aria-modal flag. Can you check?

**Comment by aneeshack4** [comment]
@saini-ankish We faced a similar issue on the Modal component and I added aria-modal="true" with PR #6944 but we still have to wait for Narrator to pick this up on future releases and provide support for it. I can do the same for Panel but first, when I tried to repro this, I was able to arrow around in the Panel component while in scan mode - the focus is not moving to the background elements: 
![image](https://user-images.githubusercontent.com/4161791/51143395-52861c00-1803-11e9-84de-f420d6df5670.png)

Is this still repro-ing for you? If so, can you take a screen capture video so we can understand what's happening?

**Comment by aneeshack4** [comment]
@saini-ankish Thank you for sending me this video - I can see it repros for you. I've created the PR with this change #7654 however I'm still not able to repro this - when I run Narrator on scan mode, I'm able to arrow throw the modal dialog that opens and when I arrow to the Save button and press enter, Narrator moves focus back to the "Open Panel" button as it should. 

I've been trying to make a screen capture video to show this to you but the Microsoft Expression Encoder software I've used so far is not working for me and I tried to use Win10's built in screen capture (WIN key + G) but it doesn't capture Narrator's blue focus rects... what screen capture software do you use?
